message_definitions/v1.0/messages.xml

```diff
@@ -2353,6 +2360,17 @@
    <field name="length" type="uint8"/>
    <field name="rtcm" type="uint8[]"/>
  </message>
+ 
+  <message name="ACINFO_LLA" id="14" link="broadcasted">
+  <field name="course" type="int16" unit="decideg"></field>
+  <field name="lat"  type="int32" unit="1e7deg" alt_unit="deg" alt_unit_coef="0.0000001"></field>
+  <field name="lon"  type="int32" unit="1e7deg" alt_unit="deg" alt_unit_coef="0.0000001"></field>
+  <field name="alt"    type="int32" unit="cm">Height above ellipsoid</field>
```

Comment:
Probably nice to add SI units as `alt_unit` so in this case `alt_unit="m"`.
If we add that to all relevant fields, it makes comparing and plotting much easier.